docs/RouteRequest.md

```diff
@@ -108,6 +110,8 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 |    [optimizeTransferWaitTime](#rd_to_optimizeTransferWaitTime)                                       |        `boolean`       | This enables the transfer wait time optimization.                                                                                  | *Optional* | `true`                   |  2.1  |
 | [transitReluctanceForMode](#rd_transitReluctanceForMode)                                             |  `enum map of double`  | Transit reluctance for a given transport mode                                                                                      | *Optional* |                          |  2.1  |
 | [unpreferred](#rd_unpreferred)                                                                       |        `object`        | Parameters listing authorities or lines that preferably should not be used in trip patters.                                        | *Optional* |                          |  2.2  |
+|    [agencies](#rd_unpreferred_agencies)                                                              |   `feed-scoped-id[]`   | TODO                                                                                                                               | *Optional* |                          |  2.0  |
+|    [routes](#rd_unpreferred_routes)                                                                  |   `feed-scoped-id[]`   | TODO                                                                                                                               | *Optional* |                          |  2.0  |
```

Comment:
Summary and is the version 2.0 ?

Reply:
Added in 22fedff9dcdd67ed49d01aca062add859626cc44